Makefile

```diff
@@ -16,6 +18,7 @@ GORELEASER_VERSION := v1.15.1
 YAJSV_VERSION := v1.4.1
 COSIGN_VERSION := v1.13.1
 QUILL_VERSION := v0.2.0
+GLOW_VERSION := v1.4.1
```

Comment:
Should this get added to the update bootstrap dependency workflow?

Reply:
good call -- updated